src/ui/views/CreateMnemonics/DisplayMnemonic.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { InfoCircleOutlined } from '@ant-design/icons';
+
+import WordsMatrix from '@/ui/component/WordsMatrix';
+
+import { StrayPageWithButton } from 'ui/component';
+import { useMedia } from 'react-use';
+import clsx from 'clsx';
+import { connectStore, useRabbyDispatch, useRabbySelector } from 'ui/store';
+import LessPalette from '@/ui/style/var-defs';
+import { fadeColor, styid } from '@/ui/utils/styled';
+
+import IconMaskIcon from '@/ui/assets/create-mnemonics/mask-lock.svg';
+import IconCopy from 'ui/assets/component/icon-copy.svg';
+import IconSuccess from 'ui/assets/success.svg';
+import { message } from 'antd';
+import { copyTextToClipboard } from '@/ui/utils/clipboard';
+
+const AlertBlock = styled.div`
+  padding: 10px 12px;
+  color: ${LessPalette['@color-red']};
+  background-color: ${fadeColor({ hex: LessPalette['@color-red'], fade: 10 })};
+`;
+
+const MnemonicsMask = styled.div`
+  background-color: ${fadeColor({ hex: '#000000', fade: 90 })};
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  backdrop-filter: blur(120px);
```

Comment:
if you use `backdrop-filter` means you have to set the mnemonics display element as children of mask element, otherwise you have to set `filter: blur(xxpx)` to mnemonics display element

Reply:
thx, fixed